src/index/TextScoring.cpp

```diff
@@ -6,6 +6,27 @@
 
 #include "index/Index.h"
 
+// ____________________________________________________________________________
```

Comment:
Functions that only appear in a .cpp file should either be marked as `static`, so
`static void logWord...` or be in the anonymous namespace 
```
namespace {
void logWordAn...
}
```
This way they never leak outside the .cpp file.